lms/djangoapps/edraak_university/views.py

```diff
@@ -85,13 +110,33 @@ def get_success_url(self):
 
     def get_context_data(self, **kwargs):
         data = super(UniversityIDView, self).get_context_data(**kwargs)
+        university_id =get_university_id(self.request.user, self.kwargs['course_id'])
+
+        if university_id:
+            terms_and_conditions_read = university_id.terms_and_conditions_read
+        else:
+            terms_and_conditions_read = False
         data.update({
             'form': self.get_form(),
             'has_valid_information': has_valid_university_id(self.request.user, self.kwargs['course_id']),
+            'form_is_disabled': self.form_disabled(self.get_course().id),
+            'terms_conditions': self.get_university_id_settings(self.get_course().id).get_terms_and_conditions(),
+            'terms_and_conditions_read': terms_and_conditions_read ,
         })
 
         return data
 
+    def form_disabled(self, course_key):
```

Comment:
This method is only used in the main view since the disabling is required for the student side.